### PR TITLE
Vulnerability detector test module refactor: `test_min_full_scan_interval`

### LINKED ISSUE #2461: Vulnerability detector test module refactor: `test_general_settings_min_full_scan_interval`
It is asked to refactor the test module named `test_general_settings_min_full_scan_interval.py`.

It is disabled for now, as it was failing or unstable, causing false positives.

## Tasks

- [x] Make a study of the objectives of the test, and what is being tested.
- [x] Refactor the test. Clean and modularizable code.
- [x] Check that the test always starts from the same state and restores it completely at the end of the test (independent of the tests previously executed in that environment).
- [x] Review test documentation, and modify if necessary.
- [x] Proven that tests **pass** when they have to pass.
- [x] Proven that tests **fail** when they have to fail.
- [x] Test in 5-10 rounds of execution that the test always shows the same result.

## Checks

- [x] The code complies with the standard PEP-8 format.
- [x] Python codebase is documented following the Google Style for Python docstrings.
- [x] The test is processed by `qa-docs` tool without errors.

### PR DESCRIPTION
**Pull request by roronoasins**
|Related issue|
|---|
|Close #2461|

## Description

This PR adds a `test_min_full_scan_interval` refactor using the new structure of the `wazuh-qa` repository and a configuration utility.

### Added

- Add new utility function for the configuration module.
- Add new monitoring functions for log events.
- Add testing module: `test_min_full_scan_interval` and data.

### Improvements

- Method to update the configuration items loaded from the template.
- Now the tier 5 checks that the full scan is performed correctly.

### Tests

- [x] `test_min_full_scan_interval` - tier 0
- [x] `test_min_full_scan_interval` - tier 5

### Outputs

<details>
<summary>All vulnerability detector tests</summary>

```
======================== test session starts ========================
platform linux -- Python 3.6.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /mnt/qa/tests/integration, configfile: pytest.ini
plugins: metadata-1.11.0, testinfra-5.0.0, html-3.1.1
collected 2589 items                                                

test_vulnerability_detector/test_feeds/test_download_feeds.py s [  0%]
sssssssssssssss                                               [  0%]
test_vulnerability_detector/test_feeds/test_invalid_type_custom_feeds.py s [  0%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [  3%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssss      [  5%]
test_vulnerability_detector/test_feeds/test_invalid_type_url_feeds.py s [  5%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [  7%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssss      [  9%]
test_vulnerability_detector/test_feeds/test_validate_feed_content.py . [  9%]
.....................................                         [ 11%]
test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py s [ 11%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 13%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 15%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 18%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss  [ 20%]
test_vulnerability_detector/test_feeds/alas/test_invalid_syntax_alas_feed.py s [ 20%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 23%]
ssssssssssssssssssssssssssssssssssssssssssssssss              [ 24%]
test_vulnerability_detector/test_feeds/alas/test_invalid_values_alas_feed.py s [ 24%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 27%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss  [ 29%]
test_vulnerability_detector/test_feeds/alas/test_missing_tags_alas_feed.py s [ 29%]
sssssssssssss                                                 [ 30%]
test_vulnerability_detector/test_feeds/archlinux/test_extra_tags_archlinux_feed.py s [ 30%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 32%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss  [ 34%]
test_vulnerability_detector/test_feeds/archlinux/test_invalid_syntax_archlinux_feed.py s [ 34%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssss        [ 36%]
test_vulnerability_detector/test_feeds/archlinux/test_invalid_values_archlinux_feed.py s [ 37%]
ssssssssssssssssssss                                          [ 37%]
test_vulnerability_detector/test_feeds/archlinux/test_missing_tags_archlinux_feed.py s [ 37%]
ssssssssss                                                    [ 38%]
test_vulnerability_detector/test_feeds/canonical/test_extra_tags_canonical_feed.py s [ 38%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 40%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss  [ 42%]
test_vulnerability_detector/test_feeds/canonical/test_invalid_syntax_canonical_feed.py s [ 42%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 45%]
sssssssssssssssssssss                                         [ 46%]
test_vulnerability_detector/test_feeds/canonical/test_invalid_values_canonical_feed.py s [ 46%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 48%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 50%]
ssssssssssssssssssssssssssssssssssssssssssssssss              [ 52%]
test_vulnerability_detector/test_feeds/canonical/test_missing_tags_canonical_feed.py s [ 52%]
ssssssssssssssssssssssssssssssss                              [ 53%]
test_vulnerability_detector/test_feeds/debian/test_extra_tags_debian_feed.py s [ 54%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 56%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss  [ 58%]
test_vulnerability_detector/test_feeds/debian/test_invalid_syntax_debian_feed.py s [ 58%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 61%]
sssssssssssssssssssss                                         [ 61%]
test_vulnerability_detector/test_feeds/debian/test_invalid_values_debian_feed.py s [ 61%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 64%]
sssssssssssssssssss                                           [ 65%]
test_vulnerability_detector/test_feeds/debian/test_missing_tags_debian_feed.py s [ 65%]
sssssssssssssssssssssssssssssssssssss                         [ 66%]
test_vulnerability_detector/test_feeds/msu/test_extra_fields_msu_feed.py s [ 66%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 68%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss  [ 71%]
test_vulnerability_detector/test_feeds/msu/test_invalid_syntax_msu_feed.py s [ 71%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssss      [ 73%]
test_vulnerability_detector/test_feeds/msu/test_invalid_values_msu_feed.py s [ 73%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss  [ 75%]
test_vulnerability_detector/test_feeds/msu/test_missing_fields_msu_feed.py s [ 75%]
ssssss                                                        [ 76%]
test_vulnerability_detector/test_feeds/redhat/test_extra_fields_redhat_feed.py s [ 76%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 78%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss  [ 80%]
test_vulnerability_detector/test_feeds/redhat/test_invalid_syntax_redhat_feed.py s [ 80%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 83%]
ssssssssssssssssssss                                          [ 83%]
test_vulnerability_detector/test_feeds/redhat/test_invalid_values_redhat_feed.py s [ 83%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 86%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 88%]
sssssssssssssssssssssssssssssssssssss                         [ 90%]
test_vulnerability_detector/test_feeds/redhat/test_missing_fields_redhat_feed.py s [ 90%]
ssssssssssssssssssssssssss                                    [ 91%]
test_vulnerability_detector/test_general_settings/test_general_settings_enabled.py . [ 91%]
ss.                                                           [ 91%]
test_vulnerability_detector/test_general_settings/test_general_settings_interval.py . [ 91%]
...........                                                   [ 91%]
test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py . [ 91%]
                                                              [ 91%]
test_vulnerability_detector/test_general_settings/test_general_settings_retry_interval.py s [ 91%]
ss                                                            [ 91%]
test_vulnerability_detector/test_general_settings/test_general_settings_run_on_start.py . [ 91%]
.                                                             [ 92%]
test_vulnerability_detector/test_providers/test_providers_disabled.py . [ 92%]
...............................                               [ 93%]
test_vulnerability_detector/test_providers/test_providers_enabled.py . [ 93%]
...............................                               [ 94%]
test_vulnerability_detector/test_providers/test_providers_multiple_providers.py s [ 94%]
sssssssssssss                                                 [ 95%]
test_vulnerability_detector/test_providers/test_providers_no_os.py s [ 95%]
ssssss                                                        [ 95%]
test_vulnerability_detector/test_providers/test_providers_os.py s [ 95%]
ssssssssssss                                                  [ 95%]
test_vulnerability_detector/test_providers/test_providers_update_from_year.py s [ 95%]
sssssssssssss                                                 [ 96%]
test_vulnerability_detector/test_providers/test_providers_update_interval.py s [ 96%]
sssssssssssssssssssssssssss                                   [ 97%]
test_vulnerability_detector/test_scan_results/test_alas_inventory_alas_feed.py s [ 97%]
s                                                             [ 97%]
test_vulnerability_detector/test_scan_results/test_alert_vulnerability_removal.py s [ 97%]
                                                              [ 97%]
test_vulnerability_detector/test_scan_results/test_archlinux_inventory_archlinux_feed.py s [ 97%]
s                                                             [ 97%]
test_vulnerability_detector/test_scan_results/test_debian_inventory_debian_feed.py s [ 97%]
s                                                             [ 97%]
test_vulnerability_detector/test_scan_results/test_macos_inventory.py s [ 97%]
s                                                             [ 97%]
test_vulnerability_detector/test_scan_results/test_msu_inventory_msu_feed.py s [ 97%]
ss                                                            [ 97%]
test_vulnerability_detector/test_scan_results/test_redhat_duplicate_vulns.py s [ 97%]
ss                                                            [ 97%]
test_vulnerability_detector/test_scan_results/test_redhat_inventory_redhat_feed.py s [ 98%]
sss                                                           [ 98%]
test_vulnerability_detector/test_scan_results/test_scan_different_cves.py s [ 98%]
sssssssssss                                                   [ 98%]
test_vulnerability_detector/test_scan_results/test_scan_nvd_feed.py s [ 98%]
sssssssssssssss                                               [ 99%]
test_vulnerability_detector/test_scan_results/test_scan_providers_and_nvd_feed.py s [ 99%]
sssssssssss                                                   [ 99%]
test_vulnerability_detector/test_scan_results/test_ubuntu_inventory_canonical_feed.py s [ 99%]
sss                                                           [ 99%]
test_vulnerability_detector/test_scan_types/test_baseline_scan_type.py s [ 99%]
                                                              [ 99%]
test_vulnerability_detector/test_scan_types/test_full_scan_type.py . [ 99%]
                                                              [ 99%]
test_vulnerability_detector/test_scan_types/test_partial_scan_type.py s [ 99%]
                                                              [ 99%]
test_vulnerability_detector/test_windows/test_cpe_indexing.py s [100%]

=========== 120 passed, 2469 skipped in 778.19s (0:12:58) ===========
```

</details>

<details>
<summary>test_min_full_scan_interval - tier 0</summary>

```
======================================== test session starts =========================================
platform linux -- Python 3.6.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /mnt/qa/tests/integration, configfile: pytest.ini
plugins: metadata-1.11.0, testinfra-5.0.0, html-3.1.1
collected 2 items                                                                                    

test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py . [ 50%]
s                                                                                              [100%]

=================================== 1 passed, 1 skipped in 23.72s ====================================

```

</details>

<details>
<summary>test_min_full_scan_interval - tier 5</summary>

```
======================== test session starts ========================
platform linux -- Python 3.6.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /mnt/qa/tests/integration, configfile: pytest.ini
plugins: metadata-1.11.0, testinfra-5.0.0, html-3.1.1
collected 2 items                                                   

test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py s [ 50%]
.                                                             [100%]

=================== 1 passed, 1 skipped in 44.60s ===================
```

</details>

### CODE REVIEW

File: deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py [hunk 1]
```diff
@@ -247,3 +251,80 @@ def check_vulnerability_scan_remove_alerts(wazuh_alerts_monitor, package, cve):
...
+        provider_name (str): Provider name of the downloaded feed.
+    """
+    check_vuln_detector_event(timeout=vd.DOWNLOAD_TIMEOUT,
+                              callback=f"The update of the '{provider_name}' feed finished",
+                              error_message=f"Could not find {provider_name} feed finished log")
+
+
+def check_vulnerability_scan_remove_vuln_package(package, cve):
+    """Check if removed vulnerable packages are reported by vulnerability detector in the alerts.
+    Args:
```
Comment (jmv74211) [inline deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py:280]:
```suggestion
    """Check if removed vulnerable packages are reported by vulnerability detector in the alerts.
    
    Args:
```

File: deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py [hunk 2]
```diff
@@ -247,3 +251,80 @@ def check_vulnerability_scan_remove_alerts(wazuh_alerts_monitor, package, cve):
...
+        package (str): Name of custom package to check. Example: 'firefox-0'
+        cve (str): Package CVE. Example: 'CVE-2019-11764'
+    """
+    check_vuln_detector_event(timeout=vd.VULN_DETECTOR_SCAN_TIMEOUT,
+                              update_position=False, callback=f"Package '{package}' not vulnerable to '{cve}'.",
+                              prefix='.*', error_message='Could not find the alert which says that the package '
```
Comment (jmv74211) [inline deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py:286]: It is necessary to pass the `update_position`  and `prefix`?

File: deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py [hunk 3]
```diff
@@ -247,3 +251,80 @@ def check_vulnerability_scan_remove_alerts(wazuh_alerts_monitor, package, cve):
...
+                                                         f"{package} is no longer vulnerable with {cve}")
+
+
+def check_vulnerability_scan_remove_vuln(package, cve):
+    """Check if removed non-vulnerable package vulnerability is reported by vulnerability detector in the alerts.
+    Args:
```
Comment (jmv74211) [inline deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py:292]:
```suggestion
    """Check if removed non-vulnerable package vulnerability is reported by vulnerability detector in the alerts.
    
    Args:
```

File: deps/wazuh_testing/wazuh_testing/tools/configuration.py [hunk 4]
```diff
@@ -642,3 +644,79 @@ def set_local_internal_options_dict(dict_local_internal_options):
...
+        test_cases_ids.append(test_case['name'])
+
+    return configuration_parameters, configuration_metadata, test_cases_ids
+
+
+def update_configuration_template(configurations, old_values, new_values):
+    """Update the configuration_template with specific values.
+
+    Args:
+        configurations (list(dict)): Actual configurations from the template.
```
Comment (jmv74211) [inline deps/wazuh_testing/wazuh_testing/tools/configuration.py:706]:
```suggestion
        configurations (list(dict)): Configuration templates.
```

File: deps/wazuh_testing/wazuh_testing/tools/configuration.py [hunk 5]
```diff
@@ -642,3 +644,79 @@ def set_local_internal_options_dict(dict_local_internal_options):
...
+        metadata_parameters = {'name': test_case['name'], 'description': test_case['description']}
+        metadata_parameters.update(test_case['metadata'])
+        configuration_metadata.append(metadata_parameters)
+        test_cases_ids.append(test_case['name'])
+
+    return configuration_parameters, configuration_metadata, test_cases_ids
+
+
+def update_configuration_template(configurations, old_values, new_values):
+    """Update the configuration_template with specific values.
```
Comment (jmv74211) [inline deps/wazuh_testing/wazuh_testing/tools/configuration.py:703]:
```suggestion
    """Update the configuration templates with specific values. Useful for setting the configuration dynamically.
```

File: tests/integration/test_vulnerability_detector/test_general_settings/data/configuration_template/min_full_scan_interval.yaml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        - name: 'syscollector'
+      elements:
+        - disabled:
+            value: 'yes'
+    - section: vulnerability-detector
+      elements:
+        - enabled:
+            value: 'yes'
+        - interval:
+            value: "20s"
```
Comment (jmv74211) [inline tests/integration/test_vulnerability_detector/test_general_settings/data/configuration_template/min_full_scan_interval.yaml:13]:
```suggestion
            value: '20s'
```

File: tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py [hunk 7]
```diff
@@ -47,162 +48,254 @@
...
-from wazuh_testing.tools.monitoring import FileMonitor
-from wazuh_testing.tools.time import TimeMachine
+import wazuh_testing.vulnerability_detector as vd
+import wazuh_testing.db_interface.agent_db as adb
+from wazuh_testing import DB_PATH
 from wazuh_testing.tools.time import time_to_seconds
+from wazuh_testing.db_interface.cve_db import get_sqlite_query_result
+from wazuh_testing.modules.vulnerability_detector import event_monitor as evm
+from wazuh_testing.modules.vulnerability_detector.utils import clean_vd_tables
+from wazuh_testing.tools import configuration
```
Comment (jmv74211) [inline tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py:69]: Sort the imports correctly, pls

File: tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py [hunk 8]
```diff
@@ -47,162 +48,254 @@
...
+
+    # Change the package version to a non-vulnerable one, to check if the full scan(next one) erases it
+    adb.update_package(package=package_names[2],
+                       version=metadata['package_version_not_vulnerable'])
+
+    # Wait for the non-vulnerable package removal event
+    evm.check_vulnerability_scan_remove_vuln_package(package_names[2], package_cves[2])
+    # Check that non-vulnerable package does not appear in vuln_cve db
+    package_2_query = get_sqlite_query_result(os.path.join(DB_PATH, '000.db'), "SELECT count(*) FROM vuln_cves "
+                                                                               f"WHERE name='{package_names[2]}';")
```
Comment (jmv74211) [inline tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py:297]: Same here

**Review by jmv74211 — CHANGES_REQUESTED** [review]

**Comment by roronoasins** [comment]
The extended behavior has been removed from this test as there is another test that checks the full scan flow. https://github.com/wazuh/wazuh-qa/pull/2540/commits/6a6cfc73a5c0e11e4d3a086dd818cb61703ca4a8

File: deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py (hunk 1 above)
Comment (roronoasins) [reply deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/1c627ab09b295462b74b4591310c2df8e20ee88f

File: deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py (hunk 2 above)
Comment (roronoasins) [reply deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/1c627ab09b295462b74b4591310c2df8e20ee88f

File: deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py (hunk 3 above)
Comment (roronoasins) [reply deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/1c627ab09b295462b74b4591310c2df8e20ee88f

File: deps/wazuh_testing/wazuh_testing/tools/configuration.py [hunk 9]
```diff
@@ -642,3 +644,79 @@ def set_local_internal_options_dict(dict_local_internal_options):
...
+
+    Args:
+        configurations (list(dict)): Actual configurations from the template.
+        old_values (list)): Values to be replace.
+        new_values (list): New values.
+
+    Raises:
+        ValueError: If the number of configurations are not the same as the number of values items to replace.
```
[the remark this answers is not in the document]
Comment (roronoasins) [reply deps/wazuh_testing/wazuh_testing/tools/configuration.py]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/f77fa8fd6da72878412ac746f689788241ef3e6f

File: deps/wazuh_testing/wazuh_testing/tools/configuration.py (hunk 4 above)
Comment (roronoasins) [reply deps/wazuh_testing/wazuh_testing/tools/configuration.py]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/f77fa8fd6da72878412ac746f689788241ef3e6f

File: deps/wazuh_testing/wazuh_testing/tools/configuration.py (hunk 5 above)
Comment (roronoasins) [reply deps/wazuh_testing/wazuh_testing/tools/configuration.py]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/f77fa8fd6da72878412ac746f689788241ef3e6f

File: deps/wazuh_testing/wazuh_testing/tools/configuration.py [hunk 10]
```diff
@@ -642,3 +644,79 @@ def set_local_internal_options_dict(dict_local_internal_options):
...
+        ValueError: If the number of values to replace are not the same.
+    """
+    if len(configurations) != len(old_values) != len(new_values):
+        raise ValueError(f"The number of configuration and values items should be the same.")
```
[the remark this answers is not in the document]
Comment (roronoasins) [reply deps/wazuh_testing/wazuh_testing/tools/configuration.py]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/f77fa8fd6da72878412ac746f689788241ef3e6f

File: tests/integration/test_vulnerability_detector/test_general_settings/data/configuration_template/min_full_scan_interval.yaml (hunk 6 above)
Comment (roronoasins) [reply tests/integration/test_vulnerability_detector/test_general_settings/data/configuration_template/min_full_scan_interval.yaml]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/e0e2ecb2f4d7638aa5d98b50aab7c7336ad8949c

File: tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py [hunk 11]
```diff
@@ -47,162 +48,254 @@
...
-    """
-    It mocks an agent with syscollector packages in a synced state
+    Finally, It cleans all in the teardown.
+
+    Args:
+        metadata (dict): Test case metadata.
     """
-    vd.insert_osinfo(agent=mock_agent_module)
+    clean_vd_tables()
+    time.sleep(0.5)
```
[the remark this answers is not in the document]
Comment (roronoasins) [reply tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py]: Solved https://github.com/wazuh/wazuh-qa/pull/2540/commits/6a6cfc73a5c0e11e4d3a086dd818cb61703ca4a8

File: tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py (hunk 7 above)
Comment (roronoasins) [reply tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py]: Solved https://github.com/wazuh/wazuh-qa/commit/6a6cfc73a5c0e11e4d3a086dd818cb61703ca4a8

**Review by jmv74211 — APPROVED** [review]

**Pull request merged**